public/app/percona/backup/components/RestoreHistory/RestoreLogsModal/RestoreLogsModal.tsx

```diff
@@ -0,0 +1,14 @@
+import { Modal } from '@percona/platform-core';
+import React, { FC } from 'react';
+
+import { ChunkedLogsViewer } from '../../ChunkedLogsViewer/ChunkedLogsViewer';
+
+import { RestoreLogsModalProps } from './RestoreLogsModal.types';
+
+export const RestoreLogsModal: FC<RestoreLogsModalProps> = ({ title, isVisible, onClose, getLogChunks }) => {
+  return (
```

Comment:
You could return the component directly here without the return statement.

Reply:
Yep 👍